README.md

```diff
@@ -471,4 +471,6 @@ For fields in `participant` such as `actor` or `itemGrants`, server will return
 - `.shardId`
 - `.stats`
 - `.titleId`
+- `.SkillTier` - Skilltier Name 
```

Comment:
Camelcase as this should match the changed method. Also remove the `-` so it follows the same convention as the others

Reply:
Fixed. It was actually a misspelling as the class method is camelcase